ejb/ejb2-connector/ejb2-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/artifacts/ESB/config/restRequests/stateful.json

```diff
@@ -1,4 +1,5 @@
 {
+
```

Comment:
If this file contain empty json object, why can't we remove this file from this connector and pass empty object in the corresponding place.